markdownhighlighter.cpp

```diff
@@ -2356,6 +2369,16 @@ bool MarkdownHighlighter::isPosInACodeSpan(int blockNumber, int position) const
     }) != rangeList.cend();
 }
 
```

Comment:
Same code as in the function above, just changed to Link. We could merge these functions